src/components/Select/src/SelectControl.vue

```diff
@@ -12,7 +12,23 @@
 			<!-- @slot Select prefix -->
 			<slot name="prefix" />
 		</span>
+		<button
+			v-if="$slots.default"
+			:class="[
+				$s.Select,
```

Comment:
Just to be a bit safer, could we create a separate class for this like `.SelectButton` you can then share the CSS with the `.Select` and more safely target the new styling just to the button.